add size and rank of nightly facto

zonglinpeng opened the pull request:
Summary: x2 scale number of testcases

Differential Revision: D68690963

mcremon-meta commented on examples/cadence/operators/facto_util.py:
```diff
@@ -18,58 +18,36 @@
 def apply_tensor_contraints(op_name: str, tensor_constraints: list[object]) -> None:
     match op_name:
-        case (
-            "sigmoid.default"
-            | "_softmax.default"
-            | "rsqrt.default"
-            | "exp.default"
-            | "mul.Tensor"
-            | "div.Tensor"
```
does that mean the removed ones use default params now?

zonglinpeng replied:
correct. they used to require stricter contraints because of kernel impl. now they should be all fixed @mcremon-meta

zonglinpeng committed:
add size and rank of nightly facto (#7962)

Summary:

x1.3 scale number of testcases (10-40 cases per op). The hardcoded numbers (size, rank, etc) are defined by statistics from test_aten_ops.py. For most ops the inputs have 1~4 dims, 1~512 of each size, -16.0~16.0 of each value. Dtype is any of float and int expecte 64 bits.

Reviewed By: hsharma35

Differential Revision: D68690963
